### PR TITLE
dashboard: update to 0319

### PR DESCRIPTION
**Pull request by HunDunDM**
Signed-off-by: Zheng Xiangsheng <hundundm@gmail.com>

### What problem does this PR solve? 

Update Dashboard to v0319

### What is changed and how it works?

Update Dashboard to v0319

### Check List 

Tests 

 - Manual test

### CODE REVIEW

File: pkg/dashboard/apiserver.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	etcdCfg, err := cfg.GenEmbedEtcdConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	dashboardCfg := &config.Config{
+		DataDir:    cfg.DataDir,
+		PDEndPoint: etcdCfg.ACUrls[0].String(),
+	}
+	dashboardCfg.ClusterTLSConfig, err = cfg.Security.ToTLSConfig()
```
Comment (HunDunDM) [inline pkg/dashboard/apiserver.go]: How do we get TiDBTLSConfig now?

Comment (breeswish) [reply pkg/dashboard/apiserver.go]: See https://github.com/pingcap/pd/pull/2246

Comment (HunDunDM) [reply pkg/dashboard/apiserver.go]: Looks like it hasn't merged yet

**Pull request merged**